salt-master/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM centos:7.6.1810
+
+MAINTAINER moonshot-platform <moonshot-platform@scality.com>
+
+# Salt version to use
+ENV SALT_VERSION 2018.3-1
+
+# salt-master console logging log level
+ENV SALT_LOG_LEVEL warning
+
+# Install saltstack repository
+RUN yum install -y https://repo.saltstack.com/yum/redhat/salt-repo-${SALT_VERSION}.el7.noarch.rpm
```

Comment:
Shouldn't we specify a full version (i.e. including patch) instead?

Reply:
Yes indeed.

Reply:
Isn't that what we have: `ENV SALT_VERSION 2018.3-1` (to be replaced with ARG)?

Reply:
No, that pins to 2018.3, not 2018.3.3.

Reply:
As they only have one release version for a given main version (eg. `2018.3`) in their repository, I don't see the point (aka, if we explicitly pull version `2018.3.3` and they update to `2018.3.4`, then it's broken on our side, and we'll have to edit docker file...) or am I missing something?

Speaking of that, I even don't think the "`-1`" should be there either (=> `ARG SALT_VERSION=2018.3`)

Reply:
See https://repo.saltstack.com/#rhel and click 'Pin to minor release'.

Reply:
https://github.com/scality/metalk8s/issues/600